02_ShowUser.php

```diff
@@ -2,112 +2,110 @@
-					<div class="w3-right" >
-					  <a href="indexV1.php" class="w3-bar-item w3-button w3-xxlarge fa fa-home"></a>
-					  <a href="playlistV1.php" class="w3-bar-item w3-button w3-xxlarge material-icons">playlist_add</a>	
-					  <a href="contact.php" class="w3-bar-item w3-button w3-xxlarge material-icons" >contact_support</a>
-					  <a href="settings.php" class="w3-bar-item w3-button w3-xxlarge material-icons">settings</a>	
-    				</div>
+	<body>
+
+		<!-- Navbar (sit on top) -->
+		<div class="w3-top">
+			<div class="w3-bar w3-grey w3-padding w3-card">
+				<img src="images/placeholder.jpg" width="40" height="40">
+				<a href="indexV1.php" class="w3-bar-item w3-button"><b>Soundwave</b> </a>
+
+				<!-- Float links to the right -->
+				<div class="w3-right" >
+					<a href="indexV1.php" class="w3-bar-item w3-button w3-xxlarge fa fa-home"></a>
+					<a href="playlistV1.php" class="w3-bar-item w3-button w3-xxlarge material-icons">playlist_add</a>	
+					<a href="contact.php" class="w3-bar-item w3-button w3-xxlarge material-icons" >contact_support</a>
+					<a href="settings.php" class="w3-bar-item w3-button w3-xxlarge material-icons">settings</a>	
 				</div>
 			</div>
-		
-		
+		</div>
+
+
 		<br>
 		<br>
 		<br>
 
 
-			<!-- side navbar for admin + php codes -->
-			<div class="w3-sidebar w3-bar-block w3-border-right" style="width:8%">
-				<a href="03_add_user(settings).php" class="w3-bar-item w3-button w3-xxlarge material-icons">person_add</a> 
-				<a href="02_show_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">visibility</a> 
-			 	<a href="04_update_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">manage_accounts</a>
-			  	<a href="05_delete_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">person_remove</a>
+		<!-- side navbar for admin + php codes -->
+		<div class="w3-sidebar w3-bar-block w3-border-right" style="width:8%">
+			<a href="03_add_user(settings).php" class="w3-bar-item w3-button w3-xxlarge material-icons">person_add</a> 
+			<a href="02_show_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">visibility</a> 
+			<a href="04_update_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">manage_accounts</a>
+			<a href="05_delete_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">person_remove</a>
 
-			</div>
-		
+		</div>
 
 
-			<!-- Title for query: show users -->
-			<div style="margin-left:8%">
-				<div class="w3-container w3-dark-grey">
-					<h2>Show User</h2>
-				</div>
+		<!-- Title for query: show users -->
+		<div style="margin-left:8%">
+			<div class="w3-container w3-dark-grey">
+				<h2>Show User</h2>
+			</div>
 
-				
-				<!-- Users php -->
-				<div class="w3-row w3-panel w3-border">
-					<div class="w3-third w3-center w3-panel">
-						<h5><user1>USERS</user1></h5>
-						  <h6>
+			<br>
+			<!-- Users php -->
+			<div class="w3-row">
+				<div class="w3-third w3-center">
```

Comment:
kept it simple by centering it as well as using the tool of w3-third to split the names from passwords